.github/workflows/tests.yml

```
@@ -0,0 +1,42 @@
+name: Tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  tests:
+    name: ${{ matrix.php }} ${{ matrix.env.COMPOSER_FLAGS }}
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-22.04
```

I think that would fix the `can't set metadata ... on shared image` error, since Ubuntu 22.04 ships with libvips 8.12.1, whereas Ubuntu 20.04 ships with libvips 8.9.1. This particular error was removed with commit https://github.com/libvips/libvips/commit/e4db74746a195b4688b5c40d0cb2da5abccc224a (available in v8.9.2).